vite.config.ts

```diff
@@ -0,0 +1,30 @@
+import { defineConfig } from 'vite';
+import react from '@vitejs/plugin-react';
+import viteTsconfigPaths from 'vite-tsconfig-paths';
+
+export default defineConfig({
+  // depending on your application, base can also be "/"
+  base: '',
+  plugins: [react(), viteTsconfigPaths()],
+  root: './',
+  build: {
+    outDir: '../build',
```

Comment:
```suggestion
    outDir: './build',
```

You're emitting the build files in the parent directory.